test/cmd.js

```diff
@@ -895,6 +895,60 @@ describe('express(1)', function () {
         .expect(404, /<h1>Not Found<\/h1>/, done)
       })
     })
+
+    describe('--view=none', function () {
```

Comment:
Probably should be moved up into the existing `--view <engine>` section and renamed to `none`